docs/reference/connector/apis/set-connector-sync-job-error-api.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[set-connector-sync-job-error-api]]
+=== Set connector sync job error API
+++++
+<titleabbrev>Set connector sync job error</titleabbrev>
+++++
+
+Sets a connector sync job error.
+
+[[set-connector-sync-job-error-api-request]]
+==== {api-request-title}
+`PUT _connector/_sync_job/<connector_sync_job_id>/_error`
+
+[[set-connector-sync-job-error-api-prereqs]]
+==== {api-prereq-title}
+
+* To sync data using connectors, it's essential to have the Elastic connectors service running.
+* The `connector_sync_job_id` parameter should reference an existing connector sync job.
+
+[[set-connector-sync-job-error-api-desc]]
+==== {api-description-title}
+
+Sets the `error` field for the specified connector sync job and sets its `status` to `error`.
```

Comment:
Maybe it's worthwhile explaining why someone would want to do this? (It's not clear to me)

Reply:
This is mainly an API for the connectors service (or any other system executing sync jobs), which need to set errors. I can add a paragraph that this is not expected to be used by end-users maybe?

Reply:
Yeah, that could be helpful.